contracts/carbon/CarbonController.sol

```diff
@@ -26,9 +26,6 @@ contract CarbonController is
     OnlyProxyDelegate,
     Utils
 {
-    // the emergency manager role is required to pause/unpause
-    bytes32 private constant ROLE_EMERGENCY_STOPPER = keccak256("ROLE_EMERGENCY_STOPPER");
```

Comment:
You should either consider explicitly revoking this role from existing accounts or documenting this role as deprecating (i.e. so that it's clear that this remains set for existing contracts and should never be reused, etc.).

Reply:
it will be revoked from existing accounts.

Reply:
As part of the upgrade script?

Reply:
nope, before the upgrade